node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -447,8 +448,7 @@ testReset = put def
 -- Legacy transaction, no longer used. For existing Db tx logs only. Now use
 -- 'removeHistoryCache', 'insertIntoHistoryCache' or 'removeFromHistoryCache'
 updateHistoryCache :: CId Wal -> [TxHistoryEntry] -> Update ()
-updateHistoryCache cWalId cTxs =
-    wsHistoryCache . at cWalId ?= txHistoryListToMap cTxs
```

Comment:
I don't think we want this, as both `master` and `cardano-sl-1.0-exchanges` have the same implementation for `updateHistoryCache`:

``` haskell
-- Legacy transaction, no longer used. For existing Db tx logs only. Now use
-- 'removeHistoryCache', 'insertIntoHistoryCache' or 'removeFromHistoryCache'
updateHistoryCache :: CId Wal -> [TxHistoryEntry] -> Update ()
updateHistoryCache cWalId cTxs =
    wsHistoryCache . at cWalId ?= txHistoryListToMap cTxs
```